modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/CustomDataSets.tsx

```diff
@@ -664,17 +667,12 @@ const CustomDataSets = ({
 				itemsActions={[
 					{
 						data: {
-							id: 'edit',
```

Comment:
@markocikos @dsanz After removing separators standalone Edit didn't appear due to a check in  [ActionsDropdown](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.tsx#L149)
As the actionLinkRenderer.js uses the first action as a fallback, it is safe to remove the `id/actionId` relation

Reply:
I've changed DSM to prevent breaking any other use case. Anyway, we could remove it.